iac-extensions/terraform/src/test/resources/checks/Azure/CertificateBasedAuthenticationCheck/api_management.tf

```diff
@@ -16,6 +16,14 @@ resource "azurerm_api_management" "example" {
 
 resource "azurerm_api_management" "example" {
   sku_name = "Consumption_" # Compliant
+
+  foo {
+    foo = bar
+  }
+
+  foo {
+    foo = bar
+  }
```

Comment:
Is that an intentional change?

Reply:
Good catch. It's a leftover from our conversation.